core/src/utils/internal/sort.ts

```diff
@@ -1,5 +1,19 @@
+/**
+ * The default comparision between two inputs, using the javascript < and > signs.
+ *
+ * @param a - the first input
+ * @param b - the second input
+ * @returns 1, 0 or -1 depending on the default compare
+ */
 export const compareDefault = (a: any, b: any) => (a < b ? -1 : a > b ? 1 : 0);
 
+/**
+ * A comparision function between DOM elements, based on `Node.compareDocumentPosition`.
```

Comment:
It can be useful to have the link : `A comparision function between DOM elements, based on [Node.compareDocumentPosition](https://developer.mozilla.org/fr/docs/Web/API/Node/compareDocumentPosition).`